debian/control

```diff
@@ -0,0 +1,42 @@
+Source: eos-data-distribution
+Section: python
+Priority: standard
+Maintainer: Philip Withnall <withnall@endlessm.com>
+Build-Depends:
+ debhelper (>= 10),
+ dh-python,
+ python-all,
+ python-pyndn,
+ python-pytest,
+ python-setuptools,
+ python-setuptools-git,
+ python3-all,
+ python3-pyndn,
+ python3-setuptools,
+ python3-setuptools-git,
```

Comment:
I wrapped and sorted it manually (the perils of doing all your development on a non-Debian machine). I’ll double-check it.